File: rxflux2-kotlin/src/main/java/com/huyingbao/rxflux2/store/RxStoreChange.java
package com.huyingbao.rxflux2.store;

import com.huyingbao.rxflux2.action.RxAction;

/**
 * Pos a new event to notify that the store has changed
 */
public class RxStoreChange {

    private String mStoreId;
    private RxAction mRxAction;

    public RxStoreChange(String storeId, RxAction rxAction) {
        this.mStoreId = storeId;
        this.mRxAction = rxAction;
    }

    public RxAction getRxAction() {
        return mRxAction;
    }

    public String getStoreId() {
        return mStoreId;
    }
}

File: rxflux2-kotlin/src/main/java/com/huyingbao/rxflux2/store/RxStoreChange.kt
package com.huyingbao.rxflux2.store

import com.huyingbao.rxflux2.action.RxAction

/**
 * Pos a new event to notify that the store has changed
 */
class RxStoreChange(val storeId: String, val rxAction: RxAction)
